src/test/javascript/spec/component/exam/manage/student-exams/student-exam-detail.component.spec.ts

```diff
@@ -24,18 +24,16 @@ import * as sinonChai from 'sinon-chai';
 import * as chai from 'chai';
 import { ReactiveFormsModule } from '@angular/forms';
 import { MockTranslateValuesDirective } from '../../../course/course-scores/course-scores.component.spec';
-import { Exercise } from 'app/entities/exercise.model';
+import { Exercise, ExerciseType } from 'app/entities/exercise.model';
```

Comment:
`'ExerciseType' is defined but never used`

Reply:
Done